include/flyft/mesh.h

```diff
@@ -1,6 +1,9 @@
 #ifndef FLYFT_MESH_H_
 #define FLYFT_MESH_H_
 
+#include <memory>
+#include "data_view.h"
```

Comment:
You need to include flyft headers with the `flyft/` prefix:
```suggestion
#include "flyft/data_view.h"

#include <memory>
```
Also, headers should come in the following order:
1. This project
2. External libraries
3. Standard library
Separate each group with a line break.